src/client/app/reducers/graph.ts

```diff
@@ -6,14 +6,16 @@ import * as moment from 'moment';
 import { TimeInterval } from '../../../common/TimeInterval';
 import { GraphAction, GraphState, ChartTypes } from '../types/redux/graph';
 import { ActionType } from '../types/redux/actions';
+import {calculateCompareTimeInterval, ComparePeriod, SortingOrder} from '../utils/calculateCompare';
 
 const defaultState: GraphState = {
 	selectedMeters: [],
 	selectedGroups: [],
 	timeInterval: TimeInterval.unbounded(),
-	barDuration: moment.duration(1, 'month'),
-	compareTimeInterval: new TimeInterval(moment().startOf('week').subtract(7, 'days'), moment()),
-	compareDuration: moment.duration(1, 'days'),
+	barDuration: moment.duration(4, 'weeks'),
+	comparePeriod: ComparePeriod.Week,
+	compareTimeInterval: calculateCompareTimeInterval(ComparePeriod.Week, moment()),
+	sortingOrder: SortingOrder.Ascending,
```

Comment:
Could you change this to `compareSortingOrder`, just in case we add sorting to another graph in the future?